src/_spec/get.js

```diff
@@ -50,6 +50,20 @@ describe('get', () => {
     assert.equal(date, 2)
   })
 
+  it('should return the week for a monday', () => {
+    const input = new Date('2015-03-04 09:08:05.023')
```

Comment:
Yeah probably. the reason for using two different years was that they have different dates in week 1.